app/router.js

```diff
@@ -7,4 +7,8 @@ export default class Router extends EmberRouter {
 }
 
 Router.map(function() {
+  this.route('events', { path: '/events'});
+  this.route('events', { path: '/event'});
+  this.route("index", { path: '/index'});
```

Comment:
Index routes are slightly differently treated than normal routes
You can refer [emberjs index routes](https://guides.emberjs.com/release/routing/defining-your-routes/#toc_index-routes) 

Basically, you don't really need to explicitly add the path to an index route

Reply:
Done!. There is only one route now that is events